fastai/tabular/model.py

```diff
@@ -7,7 +7,9 @@
 from .core import *
 
 # Cell
-def emb_sz_rule(n_cat):
+def emb_sz_rule(
+    n_cat:int # Number of categories in the data
+) -> int: # Returns embedding size
```

Comment:
```suggestion
) -> int:
```
Docment doesn't help here IMO